Week3/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The layout looks very nice and is well done, nice job on the appearance of the app overall.